### PR TITLE
docs: add service replicas

### PR DESCRIPTION
**Pull request by nunopato**
- docs: add a section for service replicas

### CODE REVIEW

File: docs/docs/platform/service-replicas.mdx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+title: 'Service Replicas'
+sidebar_position: 1
+image: /img/og/platform/service-replicas.png
+---
+
+Service Replicas is a feature that allows you to create multiple replicas of your services, enhancing availability and fault tolerance for your Nhost apps. By distributing user requests among replicas, your apps can handle more traffic and provide a better user experience. This documentation touches on the key aspects of Service Replicas in the context of the Nhost Cloud Platform.
```
Comment (dbarrosop) [inline docs/docs/platform/service-replicas.mdx:7]: should we also add the same graph as in the blog post? Maybe a link to the announcement too?

File: docs/docs/platform/service-replicas.mdx [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
```
Comment (dbarrosop) [inline docs/docs/platform/service-replicas.mdx:1]: maybe a note about pricing, that this is only supported for people with dedicated resources and that each replica adds to the total price by adding the compute

File: docs/docs/platform/service-replicas.mdx (hunk 1 above)
Comment (nunopato) [reply docs/docs/platform/service-replicas.mdx:7]: done

File: docs/docs/platform/service-replicas.mdx (hunk 2 above)
Comment (nunopato) [reply docs/docs/platform/service-replicas.mdx:1]: done.

**Review by dbarrosop — APPROVED** [review]

**Pull request merged**